specification_situation.md

```diff
@@ -1,43 +1,54 @@
 <base target="_blank">
 
-The specification of Client Hints is divided between different standardization
-bodies and different standards.  To make matters worse, since the feature does
-not yet have multiple browser implementation support, some of the specification
-for the feature lives in PRs on the relevant standards.
+The specification of Client Hints is divided between several different
+standards.  To make matters worse, since Client Hints does not yet have support
+from two implementers, some of these bits of Client Hints spec are still hidden
+away in PRs.
+
+This document attempts to provide a clear view of all of these pieces, and how
+they fit together.
 
-This document's purpose is to make it clearer how all those pieces fit
-together, and be used as a pointer for folks that want to get a better
-understanding of the current specification situation.
 
 # High-level
 
-The specification of the Client Hints infrastructure and features is divided to
-the following specifications and proposals:
+The specification of the Client Hints is divided between the following
+specifications and proposals:
 
 * IETF [Client Hints Internet-Draft](https://httpwg.org/http-extensions/client-hints.html)
-   - Defines the Client Hints infrastructure headers (`Accept-CH` and
-     `Accept-CH-Lifetime`), their format and how servers should behave with
-     regards to them.
+   - Provides the motivation for Client Hints.
+   - Defines the fundamental Client Hints infrastructure:
+      - The `Accept-CH` response header, which servers may use to advertise
+        support for certain Client Hints.
+      - The `Accept-CH-Lifetime` response header, which servers may use to ask
+        clients to remember that support.
+   - Provides both general guidelines, and formal requirements, about Client
+     Hints’ impact on caching, security, and privacy.
+   - Does *not* define any actual, particular hints – or say anything about how
+     Client Hints works in web contexts.
+
 * WHATWG HTML specification ([PR](https://github.com/whatwg/html/pull/3774))
-   - Defines the web concepts related to Accept-CH and Accept-CH-Lifetime,
-     their processing on top-level navigation responses as well as defines the
-     related caches that make sure origin persistency is maintained.
-   - Defines image related Client-Hint features: `DPR`, `Viewport-Width`,
-     `Width` and the related `Content-DPR` response header.
+   - Defines how web clients should process the `Accept-CH` and
+     `Accept-CH-Lifetime` headers sent by servers.
+   - Defines several specific Hints, which web clients may use to tell servers
+     about their viewport/device characteristics, and connection quality.
+   - Defines Feature Policies for each of the Hints. These allow first party
```

Comment:
I would assume that the respective specs define the policies. In case of image hints, which we integrated into HTML spec, this is true.. but not so for other hints.

Reply:
I included all the definitions in HTML, but you're right that it's better to define the rest in netinfo